### PR TITLE
Move to FastAPI 

### PR DESCRIPTION
**Pull request by alextran1502**
- feat: facial recognition
- clean up
- Facial recognition job is now run
- ml/add database table (#2317)
- feat(ml): add core functions for facial recognition (#2332)
- switch to fastapi
- rm gnuvicorn

### CODE REVIEW

File: machine-learning/src/main.py [hunk 1]
```diff
@@ -105,5 +96,17 @@ def run_engine(engine, path):
...
+            _model_cache[key] = pipeline(model=model, task=task)
+        else:
+            _model_cache[key] = SentenceTransformer(model)
+    return _model_cache[key]
+
+
 if __name__ == "__main__":
-    server.run(debug=is_dev, host=server_host, port=server_port)
+    uvicorn.run("main:app", host=server_host,
+                port=int(server_port), reload=True, workers=1)
```
Comment (jrasm91) [inline machine-learning/src/main.py:112]: Is reload always true, or only for dev?

Comment (alextran1502) [reply machine-learning/src/main.py:112]: Ah I forgot to put is_dev in

File: machine-learning/src/main.py [hunk 2]
```diff
@@ -105,5 +96,17 @@ def run_engine(engine, path):
     return result
 
 
+def _get_model(model, task=None):
```
Comment (jrasm91) [inline machine-learning/src/main.py:99]: Can this method be referenced above without an issue? (Before it is defined)

**Review by jrasm91 — APPROVED** [review]

File: machine-learning/src/main.py (hunk 2 above)
Comment (alextran1502) [reply machine-learning/src/main.py:99]: Yes this works fine

**Pull request merged**